### PR TITLE
Admin suppliers layout

### PR DESCRIPTION
**Pull request by ethanmills**
### Context
Just some frontend tweaks for consistency

### Changes proposed in this pull request
- Only link delivery partners
- Remove edit links
- Move lead providers to top of table

### CODE REVIEW

**Review by seb-siem-softwire — APPROVED** [review]

File: app/helpers/supplier_helper.rb [hunk 1]
```diff
@@ -2,6 +2,6 @@
 
 module SupplierHelper
   def supplier_link(supplier)
-    supplier.is_a?(LeadProvider) ? nil : edit_admin_delivery_partner_path(supplier)
+    supplier.is_a?(LeadProvider) ? supplier.name : govuk_link_to(supplier.name, edit_admin_delivery_partner_path(supplier))
```
Comment (kicferk1) [inline app/helpers/supplier_helper.rb:5]: Eh, not really a link anymore, is it? :/

File: app/views/admin/administrators/administrators/index.html.erb [hunk 2]
```diff
@@ -1,22 +1,18 @@
-<div class="govuk-grid-row">
-  <div class="govuk-grid-column-two-thirds">
```
Comment (kicferk1) [inline app/views/admin/administrators/administrators/index.html.erb:2]: Do we not want it to be two-thirds? Just making sure.

File: app/helpers/supplier_helper.rb (hunk 1 above)
Comment (kicferk1) [reply app/helpers/supplier_helper.rb:5]: Not that I have a good name. `supplier_details_element`?

Comment (amazimbe) [reply app/helpers/supplier_helper.rb:5]: I think this is just temporary. I would just keep the name

**Review by amazimbe — APPROVED** [review]

File: app/views/admin/administrators/administrators/index.html.erb (hunk 2 above)
Comment (callumacrae) [reply app/views/admin/administrators/administrators/index.html.erb:2]: I'd say that table-based pages like this should probably be full width

**Review by callumacrae — APPROVED** [review]

File: app/views/admin/administrators/administrators/index.html.erb (hunk 2 above)
Comment (ethanmills) [reply app/views/admin/administrators/administrators/index.html.erb:2]: The other table page is full width and looks better, just changing this one for consistency

**Pull request merged**